blocked-edges/4.5.1.yaml

```diff
@@ -0,0 +1,3 @@
+to: 4.5.1
+from: *
```

Comment:
All the other examples are `from: .*` i don't think this is a glob.

Reply:
ex https://github.com/openshift/cincinnati-graph-data/blob/master/blocked-edges/4.1.1.yaml

Reply:
Good catch, thanks